src/codewhisperer/commands/onAcceptance.ts

```diff
@@ -24,7 +24,12 @@ export async function onAcceptance(acceptanceEntry: OnRecommendationAcceptanceEn
      * Format document
      */
     if (acceptanceEntry.editor) {
-        const languageContext = runtimeLanguageContext.getLanguageContext(acceptanceEntry.editor.document.languageId)
+        const languageContext = runtimeLanguageContext.getLanguageContext(
+            acceptanceEntry.editor.document.languageId,
+            acceptanceEntry.editor.document.fileName.substring(
+                acceptanceEntry.editor.document.fileName.lastIndexOf('.') + 1
+            )
```

Comment:
Looks like this is repeated in multiple places, we should probably add a utility function, `getFileExtension()` if we don't have one already

Reply:
Updated!